README.md

```diff
@@ -7,3 +7,33 @@ This repo contains end-to-end tests that aim to cover the "Becoming a Teacher" s
 1. Clone this repo.
 2. `bundle install`
 3. `rake`
+
+## Work undertaken
+
+The intention of this repo was to explore end-to-end testing across the following BAT services:
+- [Publish](https://qa.publish-teacher-training-courses.service.gov.uk)
+- [Find](https://qa.find-postgraduate-teacher-training.service.gov.uk/)
+- [Apply](https://qa.apply-for-teacher-training.service.gov.uk/)
+- [Manage](https://qa.apply-for-teacher-training.service.gov.uk/)
+
+The goal was to provide testing to ensure that a course (once published), could be found by a candidate, applied for by a candidate and finally managed by a provider.
+
+Whilst we were able to write tests to met the needs of following this journey. We encountered several issues along the way, which prevented us from continuing this project at this time.
+
+## Encountered Issues
+
+1. To test these services we ran our scenarios against the QA environments of each service. Whilst some QA environments allowed us to sign in using personas, others required us to setup and sign in with DfE accounts. This required us to use notify to locate and acquire emails and links which would allow us to access the specific service.
+
+2. Due to the nature of our tests, we couldn't rely on APIs for testing that a course could be found across the services. We required visual confirmation that a course would appear to a user across the services with the correct information.
+
+3. Given that we were unable to use the IDs of specific records, it was difficult to locate if certain records were visible on a page. We had to resort to using attributes such as name, status, .etc to identify the records we were attempting to locate and test. This again became problematic, as running the tests multiple times produced duplicate records, making the scenario difficult to test.
```

Comment:
```suggestion
3. Given that we were unable to use the IDs of specific records, it was difficult to locate if certain records were visible on a page. We had to resort to using attributes such as name, status, etc. to identify the records we were attempting to locate and test. This again became problematic, as running the tests multiple times produced duplicate records, making the scenario difficult to test.
```

Reply:
resolved in [1485593](https://github.com/DFE-Digital/bat-e2e-tests/pull/6/commits/1485593b2c99dfa0c3f7b2b5354843dd85efc273)